bstrp_functions.sh

```diff
@@ -26,6 +26,24 @@ get_cmake_version(){
   CMAKE_REVISION=$(echo "$CMAKE_VERSION" | cut -d. -f3)
 }
 
+# Checks if a version is smaller than another
+compare_version() {
```

Comment:
I think something like `version_is_less_than` would be better than `compare_version`

Reply:
renamed